products/ubuntu2004/profiles/cis_level1_server.profile

```diff
@@ -0,0 +1,891 @@
+documentation_complete: true
+
+title: 'CIS Ubuntu 20.04 Level 1 Server Benchmark'
+
+description: |-
+    This baseline aligns to the Center for Internet Security
+    Ubuntu 20.04 LTS Benchmark, v1.0.0, released 07-21-2020.
+
+selections:
+    # 1 Initial Setup #
+    ## 1.1 Filesystem Configuration ##
+    ### 1.1.1 Disable unused filesystems ###
+    #### 1.1.1.1 Ensure mounting of cramfs filesystems is disabled (Automated)
+    - kernel_module_cramfs_disabled
+
+    #### 1.1.1.2 Ensure mounting of freevxfs filesystems is disabled (Automated)
+    - kernel_module_freevxfs_disabled
+
+    #### 1.1.1.3 Ensure mounting of jffs2 filesystems is disabled (Automated)
+    - kernel_module_jffs2_disabled
+
+    #### 1.1.1.4 Ensure mounting of hfs filesystems is disabled (Automated)
+    - kernel_module_hfs_disabled
+
+    #### 1.1.1.5 Ensure mounting of hfsplus filesystems is disabled (Automated)
+    - kernel_module_hfsplus_disabled
+
+    #### 1.1.1.6 Ensure mounting of udf filesystems is disabled (Automated)
+    - kernel_module_udf_disabled
+
+    #### 1.1.1.7 Ensure mounting of FAT filesystems is limited (Manual)
+    # Skip due to being Level 2
+
+    ### 1.1.2 Ensure /tmp is configured (Automated)
+    - partition_for_tmp
+
+    ### 1.1.3 Ensure nodev option set on /tmp partition (Automated)
+    # Needs rule
```

Comment:
Just as an FYI, these and others in this subsection are ones I was talking about w.r.t. `_optional` form; we can introduce them upstream in a future PR.